packages/eslint-custom-rules/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "eslint-plugin-no-tailwind-colors",
```

Comment:
would consider adding this to the existing eslint-config package unless there's an explicit reason to add another one.

Reply:
tried to look more into this, but couldn't make it work from inside the eslint-config package. Do you have a suggestion?